cli/configuration.md

```diff
@@ -103,23 +103,21 @@ For details on configuring custom authentication, please see
 
 ## Single Sign-On Authentication (SAML, OAuth 2.0)
 
-When using a SSO provider for authentication the `Username`, `Password`, and
-`Email` that you are prompted for are not used. We recommend providing the following:
+When using a SSO provider for authentication you need to validate your session with SSO provider.
 
-```
-Username: sso
-Password: sso
-Email: (this IS public) sso@example.com
-```
-
-_Note: newer versions of the npm CLI will eliminate the need for entering a `Username`/`Password`/`Email`
-  when SSO is used._
+Our new feature `--auth-type` will allow to validate session with SSO provider during login.
```

Comment:
Since this feature won't be new for long (and we'll probably not update docs too frequently) let's change this to:

`The flag --auth-type allows you to to delegate to a SSO provider during login`